lib/Sema/DerivedConformanceEquatableHashable.cpp

```diff
@@ -196,7 +196,8 @@ deriveEquatable_enum_eq(TypeChecker &tc, Decl *parentDecl, EnumDecl *enumDecl) {
   //   case A, B, C
   //
   //   @derived
-  //   func ==(a: SomeEnum<T...>, b: SomeEnum<T...>) -> Bool {
+  //   @_implements(Equatable, ==(_:_:))
+  //   func _DerivedEnumEquals(a: SomeEnum<T...>, b: SomeEnum<T...>) -> Bool {
```

Comment:
Drive-by thought:

Wouldn't it be neat to have this be consistent with the "__Anonymous_field" and "__Unnamed_union" terminology introduced in #6935? (I guess it'd be something like "__Derived_conformance_enum_equals".)